### PR TITLE
Add es-mda example

### PR DESCRIPTION
**Pull request by ajikmr**
Added section for ES-MDA based update in two example python files.

### CODE REVIEW

**Comment by dafeda** [comment]
This looks good to me.
I took the liberty of pushing two commits, hope you don't mind.
We like to use Jupytext to pair `.py` and `.ipynb` files, which I've done in one of the commits.
Personally I prefer to do these sorts of things in a notebook and sync to a python file, but I know others prefer the other way around.

Also, please squash your commits that have the same title.

**Comment by ajikmr** [comment]
I would need help with git squash. I found this instruction on internet to squash last three commits. Please confirm if this is what needs to be done at my end.
"git reset --soft HEAD~3 &&
git commit"

I am not able to see the *.ipynb files. That is why I did not add those files earlier. For me, 'docs/source' directory only has *.py files.

**Comment by dafeda** [comment]
The way I like to squash commits is to use interactive rebase.
So something like this:

`git rebase -i HEAD~3`

It's worthwhile to read up a bit on rebase I think.

As for the `ipynb` files, they were indeed missing which was a mistake. 
I have added them here to fix that.

**Comment by ajikmr** [comment]
Ok. I see *.ipynb files now.

**Comment by ajikmr** [comment]
I have fixed spacing in es-mda example in *.ipynb files < * sqrt(weight * length)>. 
I am not able to rebase properly. If I do rebase to merge three previous commits, then *.ipynb files stop showing.

**Comment by dafeda** [comment]
> I have fixed spacing in es-mda example in *.ipynb files < * sqrt(weight * length)>. I am not able to rebase properly. If I do rebase to merge three previous commits, then *.ipynb files stop showing.

I can't see what you updated in your latest commit since you did not include the `.py` files.
Remember that we are using Jupytext to sync notebooks with plain scripts.
Here's how they are synced:

Screenshot 2022-09-27 at 13 20 28

**Comment by ajikmr** [comment]
Line #193 in Polynomial.py and line #179 in Oscillation.py have proper spacing < * sqrt(weight * length)>. But same lines in *.ipynb were not showing spacing. So I made this commit.
If these files are generated automatically then I should revery the last commit #[169b78c]. Please confirm.

**Comment by dafeda** [comment]
You can just soft reset last commit, sync with Jupytext and push both notebook and python file.
By the way, we are also using `black` to format code.

**Comment by ajikmr** [comment]
I made no changes in *py files as those files had proper spacing. And *.ipynb files are generated using Jupytext. Hence, last commit is reverted.

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,160 @@
-conanfile.txt
-conanbuildinfo.txt
-conanbuildinfo.args
```
Comment (eivindjahren) [inline .gitignore:3]: Please don't remove these from the gitignore as they are generated by the build.

**Review by eivindjahren — CHANGES_REQUESTED** [review]

**Comment by ajikmr** [comment]
*.py files are fine as of now and required changes have been made. 

Can somebody generate the associated *.ipynb file as it is done using jupytext. *py and *.ipynb file association is same as it was there earlier in 'ert' repository.

**Comment by eivindjahren** [comment]
I created a separate PR with just the change requested, without changing the current setup of gitignore and generation of notebooks. We can still discuss notebooks and gitignores separately.

**Comment by ajikmr** [comment]
Yes. It looks fine to me.

**Comment by eivindjahren** [comment]
> Yes. It looks fine to me.

Then I would like to thank you so much for your contribution and being patient with our process! I have merged the other PR now. As I did some minor fixing to get our CI to work, I am listed as co-author of the commit. Hopefully, if you would like to contribute again, we will be more prepared to handle your contribution quickly :) .

**Comment by ajikmr** [comment]
Sure @eivindjahren . Thanks for assisting with PR. And thanks for having patience with me as I am also learning few things about making contributions using git.

**Pull request closed without merging**